orb_models/forcefield/forcefield_heads.py

```diff
@@ -133,7 +133,7 @@ def forward(
             node_features, batch.n_node, reduction=self.node_aggregation
         )
         pred = self.mlp(input)
-        return pred
+        return pred.squeeze(-1)
```

Comment:
This shouldn't be necessary, I think? In general a single graph input should have a `(1,1)` shape here, because we assume things have a batch dimension.